webapp/crashstats/settings/base.py

```diff
@@ -11,13 +11,14 @@
 import re
 import socket
 
-from decouple import config, Csv
+from everett.manager import ConfigManager, parse_bool
```

Comment:
Now that we're using Everett for configuration, we can use `automoduleconfig` to auto-document configuration.

We need to add something this to `docs/configuration.rst` in the webapp section:

```rest
.. automoduleconfig:: crashstats.settings.base._config
   :show-table:
   :hide-name:
   :case: upper
```

Note the configuration variable name will depend on what you name the configuration variable in line 21.